### DISCUSSION TITLE
Authetlia withCentreon

### DISCUSSION THREAD

**Discussion by g4fa** [authelia/authelia#7347] [Q&A] [answered]
Hi

is someone success Authelia with Centreon. ?

Thanks for your help 
Fabien

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Hello Fabien,

Yes, there has been a discussion regarding configuring Authelia with Centreon on the Authelia GitHub discussions page. It appears that someone encountered an issue with the "Request for connection token to external provider has failed" error. The problem might be related to the configuration of redirect URIs, which need to be identical at both the authorization and token endpoints to comply with the specification.

For more details, you can refer to the discussion [here](https://github.com/authelia/authelia/discussions/6466).

If you are experiencing a similar or different issue, could you please provide more specific details about the error or problem you are facing? This will help in providing more targeted assistance.

Thank you!

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by james-d-elliott** [comment, marked as answer]
Since no one has tried yet maybe post what you've tried and what the result was.
